drivers/kodi/driver.js

```diff
@@ -4,6 +4,12 @@
 var KodiWs = require('node-kodi-ws')
 var Fuse = require('fuse.js')
 var Utils = require('../../libs/utils')
+const http = require('http');

```

Comment:
Try to use the KodiWs instead of http lib to have 1 way of communicating with Kodi